<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagingWidget</class>
 <widget class="QWidget" name="PagingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PagingWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="SpacerLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex1">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex2">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex3">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex4">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex5">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex6">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex7">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonEllipsis">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>â€¦</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex99">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>99</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonIndex100">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	border:1px solid rgb(180,180,180);
	color: rgb(80,80,80);
	font-size:15px;
}
QPushButton:checked
{
	border-radius:5px;
	background-color: rgb(237, 113, 46);
	color: rgb(255,255,255);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="LabelGo">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: rgb(80,80,80);
	font-size:15px;
}</string>
     </property>
     <property name="text">
      <string>è·³è½¬è‡³</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="SBoxIndex">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox
{
	font-size:15px;
	border:1px solid rgb(180,180,180);
}
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PButtonGo">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:5px;
	background-color: rgb(180, 180, 180);
	color: rgb(80,80,80);
	font-size:15px;
}
</string>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="SpacerRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
